__tests__/Unit/Components/Tasks/TaskLevelMap.test.tsx

```diff
@@ -49,15 +61,14 @@ describe('TaskLevelMap', () => {
 
     it('renders a list of task tags with remove button when shouldEdit and isUserAuthorized are true', () => {
         const deleteTaskTagLevel = jest.fn();
-        renderWithProviders(
-            <isUserAuthorizedContext.Provider value={true}>
-                <TaskLevelMap
-                    taskTagLevel={taskTagLevel}
-                    shouldEdit={true}
-                    itemId={'1'}
-                    deleteTaskTagLevel={deleteTaskTagLevel}
-                />
-            </isUserAuthorizedContext.Provider>
+
+        render(
+            <TaskLevelMap
+                taskTagLevel={taskTagLevel}
+                shouldEdit={true}
+                itemId={'1'}
+                deleteTaskTagLevel={deleteTaskTagLevel}
+            />
         );
 
         const tagElements = screen.getAllByTestId('tag-name');
```

Comment:
Shouldn't test for if user is not authorized be done as well?

Reply:
I will be addressing this issue in this ticket

https://github.com/Real-Dev-Squad/website-status/issues/670